curvefs/proto/mds.proto

```diff
@@ -146,6 +146,24 @@ message AllocateS3ChunkResponse {
 }
 
 
+message StatMetadataSpaceRequest{
+}
+
+message StatMetadataSpaceResponse{
```

Comment:
Mds doesn't have any metaserver space info. You should add an interface between metaserver and mds.